Upstream: No
Reason: fix crash on right clicking title bar
Source: bugzilla.gnome.org (bug #614592)
--- 
--- metacity-2.30.0/src/core/core.c	2010-03-31 21:37:39.000000000 +0100
+++ metacity-2.30.0/src/core/core.c	2010-04-01 15:59:26.000000000 +0100
@@ -520,7 +520,7 @@ meta_core_get_menu_accelerator (MetaMenu
   switch (menu_op)
     {
     case META_MENU_OP_NONE:
-      name = "none";
+      /* No keybinding for this one */
       break;
     case META_MENU_OP_DELETE:
       name = "close";
